benchmarks/viscoelastic_beam_modified/10km_opentopbot.prm

```diff
@@ -0,0 +1,92 @@
+# This parameter file modifies the 2D benchmark 'viscoelastic_bending_beam'.
+# It has been modified to model a much larger beam, and to have an open top
+# and bottom boundary to allow the inelastic, low viscosity fluid surrounding the beam to 
+# be more easily displaced while the beam flexes.
+#
+# The benchmark examines bending of a viscoelastic beam surrounded by a less dense
+# and viscous (inelastic) fluid. Gravitational forces drive initial bending 
+# (elastic strain) of the beam. The analytic solution is from Geodynamics by 
+# Turcotte and Schubert. Models have been tested for deflections that are no greater 
+# than half the thickness of the beam. More flexure could potentially lead to a breakdown
+# of the fit since the analytic solution assumes small deflection relative to the thickness. 
+
+include $ASPECT_SOURCE_DIR/benchmarks/viscoelastic_bending_beam/viscoelastic_bending_beam.prm
+
+# Global parameters
+set Start time                             = 0
+set End time                               = 200e3
+set Use years in output instead of seconds = true
+set Maximum time step                      = 1e3
+set Output directory                       = 10kmh_10drho
+
+# Model geometry (240x160 km, 1 km spacing)
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 120
+    set Y repetitions = 80
+    set X extent      = 240e3
+    set Y extent      = 160e3
+  end
+end
+
+# Mesh refinement
+subsection Mesh refinement
+  set Initial adaptive refinement        = 3
+  set Initial global refinement          = 0
+  set Time steps between mesh refinement = 1
+  set Strategy                           = viscosity
+end
+
+# Traction Boundary Conditions
+subsection Boundary traction model
+  set Prescribed traction boundary indicators = top: initial lithostatic pressure, bottom: initial lithostatic pressure
```

Comment:
Good to know these types of models are stable with initial lithostatic pressure BCs on both the top and bottom!